lib/active_fedora/filter/association.rb

```diff
@@ -21,6 +21,11 @@ def concat(records)
       extending_from.concat(records)
     end
 
+    def ids_reader
+      load_target
```

Comment:
This is going to be really slow. But I don't really see anyway around it until we put the filter logic into solr.